src/content/docs/change-tracking/change-tracking-introduction.mdx

```diff
@@ -36,27 +36,27 @@ Change tracking has a number of features that help you gauge the effects of chan
 
 Change tracking goes far beyond simple deployment tracking for APM services. You can also track changes in mobile apps, infrastructure hosts, workloads, and more.
 
-Note that if you're using the old REST v2 APM deployments API, you need to migrate to our GraphQL API to start capturing changes on [entity types](/docs/new-relic-solutions/new-relic-one/core-concepts/what-entity-new-relic/#find) besides APM services. If you switch to change tracking with the GraphQL API, you'll be able to leverage our current Jenkins and GitHub Actions integrations, as well as any future CI/CD integrations.
+Note that if you're using the old REST v2 APM deployments API, you need to migrate to our GraphQL API to start capturing changes on [entity types](/docs/new-relic-solutions/new-relic-one/core-concepts/what-entity-new-relic/#find) besides APM services. If you switch to change tracking with the GraphQL API, you'll be able to leverage our current [Jenkins](/docs/change-tracking/ci-cd/change-tracking-jenkins) and [GitHub Actions](/docs/change-tracking/ci-cd/change-tracking-github-actions) integrations, as well as any future CI/CD integrations.
 
 Our goal is to provide full-platform coverage for markers on charts, but some of the New Relic curated experiences don't yet feature interactive markers on all charts. In the meantime, you can still record changes on relevant entity types and build your own experience around that type using a custom dashboard.
 
 ### Quickly answer the question “What changed?” [#what-changed]
 
 Once you start [capturing deployments](#start-tracking) and other changes to entities in your system, you'll automatically see interactive markers drawn on related charts in APM, mobile, browser, service level management, custom dashboards, and elsewhere. You'll also see entity sidebar options related to change tracking (for example, **Deployments**) popping up across New Relic.
 
-### Lower your mean time to repair (MTTR) [#mttr]
+### Lower your mean time to resolution (MTTR) [#mttr]
 
 Change tracking has powerful interfaces to help you quickly understand the impact a change has had on your system. Here are some tasks change tracking can help with:
 
-* Want to see how many regressions or new error groups we detected since a change?
+* Want to see how many regressions or new error groups we detected since a change? For more about regressions, see [errors inbox](/docs/errors-inbox/errors-inbox).
```

Comment:
Might be helpful to link to a specific section in this errors inbox doc

Reply:
@paperclypse I added this anchor: /docs/errors-inbox/errors-inbox/#status